### ISSUE TITLE
Doesn't works BT Audio after updating from 9.5 to 9.6 version

### ISSUE THREAD

**Issue by dedsecproyt** [naz664/crDroid_patches#24]
Today i updates to CrDroid 9.6 and noticed that the sound from bluetooth does not work, I tried all the methods, but I only managed to display the headphones in the selection and on the sound panel

P.S On crdroid 9.5 audio works with pph settings + misc + "use system BT HAL" and with "use alternate way to detect headset"

**Comment by naz664** [comment]
Does it works on latest td ci build ?
https://github.com/TrebleDroid/treble_experimentations/releases/tag/ci-20230706

**Comment by dragbeadle** [comment]
I have the same problem as the TS. On version 9.6 of your ROM my bluetooth headset is connected, but is not used to output audio. I cannot comment on version 9.5 as I have not used that particular version. Would be willing to try it though if that would be any use.

> https://github.com/TrebleDroid/treble_experimentations/releases/tag/ci-20230706

In this ^ version I have no problems, using the settings as mentioned by the TS.

**Comment by naz664** [comment]
What's TS ?
You mean td as trebledroid ?

**Comment by dragbeadle** [comment]
TS = Topicstarter, or in this case, [dedsecproyt](https://github.com/dedsecproyt)

I mean to say that I also experience the problem described in this bug report.

**Comment by dragbeadle** [comment]
Update: tried a few different versions. 

Bluetooth audio _does_ work in these:
https://github.com/TrebleDroid/treble_experimentations/releases/tag/ci-20230706
Your [crDroid-9.5 v2023.05.16](https://github.com/naz664/crDroid_gsi/releases/tag/v2023.05.16) build

It does _not_ work in these:
Your [SparkOS-13.5.5 v2023.03.20](https://github.com/naz664/SparkOS_gsi/releases/tag/v2023.03.20) build (may be too old?)
Your [crDroid-9.6 v2023.07.07](https://github.com/naz664/crDroid_gsi/releases/tag/v2023.07.07) build

I seem not to be able to boot your ArrowOS build, so didn't test that.

**Comment by naz664** [comment]
Try this build and check if bt works 
https://github.com/naz664/Test/releases/tag/v2023.07.17-1

**Comment by dragbeadle** [comment]
Just tried the build you suggested and for me the problem is not solved. 

Thank you for your effort in trying to squash this bug! I very much hope you'll be able to fix this eventuallly. Apart from BT audio, this ROM has everything I need for a daily driver. 

Please let me know if there is anything else you'd like me to try or test.

**Comment by naz664** [comment]
Users having bt issues should be fixed on this build.
https://github.com/naz664/crDroid_gsi/releases/tag/v2023.07.21
If not, try enabling systba wide bt hal on treble settings and do a reboot.

**Comment by dragbeadle** [comment]
The issue has been solved, I'm happy to report. Thank you very much! I'm gonna try your ROM as a daily driver now. 

Also, my s22 got stuck at boot and the version you recommend for the A53 solved it. I figured the Samsung S22 and A53 are pretty similar. Can I assume that you're going to include the bootfix in future builds?

Edit: seems fingerprint is not working now. Should I open a new issue for that?
